documentation/book/proc-deploying-cluster-operator-kubernetes-to-watch-multiple-namespaces.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// assembly-cluster-operator.adoc
+
+[id='deploying-cluster-operator-kubernetes-to-watch-multiple-namespaces{context}']
+= Deploying the Cluster Operator to watch multiple namespaces
+
+.Prerequisites
+
+* Modify the installation files according to the namespace the Cluster Operator is going to be installed in.
++
+[source, subs="+quotes"]
+----
+sed -i 's/namespace: .\*/namespace: _<my-namespace>_/' examples/install/cluster-operator/*RoleBinding*.yaml
+----
+
+.Procedure
+
+. Modify the file `examples/install/cluster-operator/050-Deployment-strimzi-cluster-operator.yaml` and in the environment variable `STRIMZI_NAMESPACE` list all namespaces or projects where Cluster Operator should watch for resources.
```

Comment:
I think we need to parameterize `{namespace}`. Also for this to make proper sense it should be something like "Kubernetes namespaces or OpenShift projects" .

Reply:
OpenShift is using namespaces. We are using namespaces. We are not using any of the _project_ functionality. I do not think we need to push this too much.

Reply:
Ok, I decided to do it at the end. Still not sure this is needed, but whatever.